src/client/features/interactions/index.js

```diff
@@ -26,14 +43,18 @@ class Interactions extends React.Component {
       },
       id:'',
       loading: true,
+      catagories: new Map (),
+      buttons:new Map([['Binding',false],
+        ['Phosphorylation',false],
+        ['Expression',false]
+      ]),
```

Comment:
Double arrays are confusing.  What are these values supposed to represent? An object would be clearer.